tests/i18n/tests.py

```diff
@@ -405,6 +405,22 @@ def test_template_tags_pgettext(self):
             rendered = t.render(Context())
             self.assertEqual(rendered, '2 andere Super-Ergebnisse')
 
+            # Using 'force_escape'
```

Comment:
I think this could be a separate test method. How about a more realistic usage example too?